src/main/java/pl/pjwstk/woloappapi/utils/EventMapper.java

```diff
@@ -113,7 +115,8 @@ public Event.EventBuilder toEvent(EventRequestDto dtoEvent){
                 .description(dtoEvent.getDescription())
                 .isPeselVerificationRequired(dtoEvent.isPeselVerificationRequired())
                 .isAgreementNeeded(dtoEvent.isAgreementNeeded())
-                .imageUrl(dtoEvent.getImageUrl());
+                .imageUrl(dtoEvent.getImageUrl())
```

Comment:
bardzo sie cieszę, że tu jest builder. Niestety widzę, że w `mapShiftToShiftDto(` nie ma. Warto to ujednolicić i iść w stronę builderów lombokowych. To przydatne rozwiązanie. Przyspiesza kodowanie.